e2e-tests/insurance/cypress/e2e/journeys/your-buyer/change-your-answers/change-your-answers-trading-history.spec.js

```diff
@@ -206,4 +206,42 @@ context('Insurance - Your buyer - Change your answers - Trading history - As an
       });
     });
   });
+
+  describe(`changing ${TRADED_WITH_BUYER} from yes to no`, () => {
```

Comment:
Can we move this test into it's own E2E test? We have this pattern in other places, easier for debugging and reduces file size, e.g `change-your-answers-pre-credit-period-yes-to-no.spec`